### PR TITLE
Menu changes

### PR DESCRIPTION
**Pull request by Ashnaali3255**
**Description**
For this milestone of your portfolio website, you will implement the mobile menu using your recently acquired JavaScript knowledge.

SECOND IMPORTANT NOTE: Remember to reach out to Student Success if you are experiencing difficulties with your partner. Difficulties might range from not hearing from your partner, your partner not having a proper working environment, or your partner not following up with agreements on finishing projects. Student Success is here to support you and we can better help you when you reach out early in the week.

General requirements
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
Make sure that you used correctly [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
HTML/CSS & JavaScript requirements
Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
Project requirements
IMPORTANT NOTE: For each Microverse project, you need to create a separate pull request. [Read how to work with multiple pull requests in your one repository in the most efficient way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md).

For this project, you should add the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

In mobile, implement the following features:

When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
Mobile menu

We included a screenshot of Template 5 for reference, but you should follow the template that you chose.

You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -17,13 +17,71 @@ header nav {
...
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 20px;
+  text-decoration: none;
+  width: 250px;
+  height: 20px;
+  color: #6070ff;
+  display: flex;
+}
```
Comment (leonardodiasb) [inline style.css]: - [ ] Your mobile menu looks good, but the width of this element is making it possible for the user to scroll horizontally, leaving whitespace on the side and making your page not look professional. Kindly remove this extra width to make the app look like the Figma template.
 - Your design:
 - 
image

File: index.html [hunk 2]
```diff
@@ -17,13 +17,20 @@
...
+          <div class="desktop-ul">
+            <div class="desktop-link"><a href="#Portfolio">Portfolio</a></div>
+            <div class="desktop-link"><a href="#About">About</a></div>
+            <div class="desktop-link"><a href="#Contact">Contact</a></div>
+          </div>
+          <div class="menu-icon">
+            <div class="item-closed"><a href="#" class="closing"><i class="fa fa-times" aria-hidden="true"></i></a></div>
+            <div class="mobile-nav"><a href="#Portfolio" class="closing">Portfolio</a></div>
+            <div class="mobile-nav"><a href="#About" class="closing">About</a></div>
+            <div class="mobile-nav"><a href="#Contact" class="closing">Contact</a></div>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Unfortunately, your menu links are not redirecting the user to the correct part of the page. Kindly make sure the `href` attribute value is correct and that the sections on the page have the same `id` values.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Ashnaali3255, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] Please do not copy the project requirements page on your PR description. The Pull Request should have a short summary describing the introduced changes in more detail, and it should be written by you. The project requirements are written for the student, no for the code reviewer that is reviewing this PR and it also has a lot of topics that should not be present in this PR description. Kindly rewrite it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Ashnaali3255) [reply index.html]: @leonardodiasb  Can you help me I don't know why it's not working I tried a lot but all in vain.

Comment (prolajumokeoni) [reply index.html]: Hi, I just cloned your repo, and when I click on the links they redirect to the correct section, do you still have any thing you are not clear about?

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Please move all images to an assets folder this will make your files easier to manage

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**